docs/api-reference/react/deckgl.md

```diff
@@ -117,6 +119,15 @@ If a certain view id is used in both JSX views and the `views` prop, the view in
   <DeckGL />
 ```
 
+#### JSX Widgets
+
+It is possible tp use JSX syntax to create deck.gl widgets as React children of the `DeckGL` React components, instead of providing them as ES6 class instances to the `widgets` props.
```

Comment:
```suggestion
It is possible to use JSX syntax to create deck.gl widgets as React children of the `DeckGL` React components, instead of providing them as ES6 class instances to the `widgets` props.
```

Should link to your new contents about React widgets...

Reply:
Can do. I should probably add small pages in this sub-module for each react-wrapped widget as well for discoverability...

Reply:
Actually, I'd prefer not to maintain 2x doc pages if we don't have to. Instead, I've added an overview page to the react module with the general import pattern. Thoughts?